File: files/problems/c/power-of-two.c
bool isPowerOfTwo(int n) {
    if (n == 0) return false;
    while (n > 1 && (n & 1) == 0) 
        n >>= 1;
    return n == 1;
}

int main() {
    printf("%d", isPowerOfTwo(16));
    return 0;
}

File: leetcode/problems/js/power-of-two.js
// by @codeAbinash
// Time : O(1)
// Space : O(1)

/**
 * @param {number} n
 * @return {boolean}
 */
var isPowerOfTwo = function (n) {
   return n > 0 && (n & n - 1) === 0
};